spec/checkPullRequestReviewSpec.js

```diff
@@ -1,11 +1,8 @@
 // Copyright 2020 The Oppia Authors. All Rights Reserved.
-//
```

Comment:
Unneeded change.

Reply:
Correcting it in the latest commit Sir.Thanks.

Reply:
No, it wasn't fixed.

Reply:
fixed them in the later commits. Sorry for Late reply.

Reply:
Still not fixed. Please always verify your changes through the GitHub UI.

Reply:
@vojtechjelinek Could you check it now and please let me know if it is Okay now. Thanks.